app/api/notifications.py

```
@@ -0,0 +1,517 @@
+        )
+    
+    await db.delete(device_token)
+    await db.commit()
+
+
+# Admin endpoints for templates (require admin role)
+
+@router.post("/templates", response_model=NotificationTemplateResponse)
+async def create_notification_template(
+    template_data: NotificationTemplateCreate,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.13.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)